webapp/actions/global_actions.jsx

```diff
@@ -42,8 +42,7 @@ export function emitChannelClickEvent(channel) {
         );
     }
     function switchToChannel(chan) {
-        AsyncClient.getChannels(true);
-        AsyncClient.getMoreChannels(true);
+        AsyncClient.getChannelsUnread(true);
```

Comment:
Is this really something that needs to happen every time we switch channels? Shouldn't this update though the websocket?